src/server/load_statistics/linux.rs

```diff
@@ -116,14 +116,14 @@ mod tests {
         let mut stats = ThreadLoadStatistics::new(2, &thread_name, Arc::clone(&load));
         let start = Instant::now();
         loop {
-            if (Instant::now() - start).as_millis() > 100 {
+            if (Instant::now() - start).as_millis() > 200 {
                 break;
             }
         }
         stats.record(Instant::now());
-        match load.load() {
-            80...100 => {}
-            e => panic!("the load must be heavy than 80, but got {}", e),
+        let cpu_usage = load.load();
+        if cpu_usage < 80 || cpu_usage > 110 {
```

Comment:
Because I guess it's not very accurate for load statistics. When I run the test case with a `println` to show the cpu usage, the most results are about `99`, but some results are beyond `100`, reaching `103` or something similar.
So I think we can use `110` as the ceil to stable the case.

Reply:
if that, only `cpu_usage < 80` is okay